extensions/parquet/table/src/main/java/io/deephaven/parquet/table/transfer/IntTransfer.java

```diff
@@ -0,0 +1,102 @@
+/**
+ * Copyright (c) 2016-2023 Deephaven Data Labs and Patent Pending
+ */
+package io.deephaven.parquet.table.transfer;
+
+import io.deephaven.chunk.IntChunk;
+import io.deephaven.chunk.attributes.Values;
+import io.deephaven.engine.rowset.RowSequence;
+import io.deephaven.engine.table.ChunkSource;
+import io.deephaven.engine.table.ColumnSource;
+import io.deephaven.util.QueryConstants;
+import org.apache.parquet.column.statistics.IntStatistics;
+import org.apache.parquet.column.statistics.Statistics;
+import org.jetbrains.annotations.NotNull;
+
+import java.nio.Buffer;
+import java.nio.IntBuffer;
+
+public class IntTransfer implements TransferObject<IntBuffer> {
+
+    private final ColumnSource<?> columnSource;
+    private final ChunkSource.GetContext context;
+    private final IntBuffer buffer;
+    private IntChunk<? extends Values> chunk;
+    private int minValue = QueryConstants.NULL_INT;
+    private int maxValue = QueryConstants.NULL_INT;
+
+    public IntTransfer(
+            @NotNull final ColumnSource<?> columnSource,
+            final int targetSize) {
+        this.columnSource = columnSource;
+        this.buffer = IntBuffer.allocate(targetSize);
+        context = columnSource.makeGetContext(targetSize);
```

Comment:
This is strictly inferior, in my opinion, for int, long, float, double, and Boolean. We seem to have standardized on the `getChunk` and copy implementation, instead of just directly filling the backing array for our chunk and our buffer at the same time. What's the upside?

I *do* really like that we're allocating the buffer and chunks inside the constructor now.